scripts/run_model.py

```diff
@@ -1,63 +1,18 @@
 import logging
-import os
-import shutil
 import sys
 
-# These have to be before we do any import from keras.  It would be nice to be able to pass in a
-# value for this, but that makes argument passing a whole lot more complicated.  If/when we change
-# how arguments work (e.g., loading a file), then we can think about setting this as a parameter.
-import random
-import numpy
-random.seed(13370)
-numpy.random.seed(1337)
-
-# pylint: disable=wrong-import-position
-
-# Unless I find a better way to do this, we'll use ConfigFactory to read in a parameter file.
-# ConfigFactory.parse_file() returns a ConfigTree, which is a subclass of OrderedDict, so the
-# return value actually matches the type our code expects, anyway.
-import pyhocon
-
-sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
-from deep_qa.common.checks import ensure_pythonhashseed_set, log_keras_version_info
-from deep_qa.common.params import Params, replace_none
-from deep_qa.common.tee_logger import TeeLogger
-from deep_qa.models import concrete_models
-from keras import backend as K
+from deep_qa import run_model
+from deep_qa.common.checks import ensure_pythonhashseed_set
 
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
+
 def main():
     if len(sys.argv) != 2:
-        print('USAGE: run_model.py [param_file]')
+        print('USAGE: run.py [param_file]')
```

Comment:
You didn't change the name of this script.  I'd probably actually keep this one as `run_model.py`, matching `run.run_model`.  But maybe that's too confusing...  Feel free to pick one, but make sure you've fixed it everywhere - various places in this PR have this as both `run.py` and `run_model.py`.